raft_consensus/protocol.nim

```diff
@@ -0,0 +1,45 @@
+# nim-raft-consesnsus
+# Copyright (c) 2023 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+                        #                                       #
+                        #   RAFT Messages Protocol definition   #
+                        #                                       #
+import types
+
+type
+    # RAFT Node Messages definitions
+    RAFTMessageOps* = enum
+        REQUEST_VOTE        = 0,
```

Comment:
Please re-format these files to conform to the EditorConfig configuration present in this repo (you must use 2 spaces for indentation).